examples/workflows/plot_simulate_somato.py

```diff
@@ -159,11 +159,9 @@
 # proximal drive. In order to understand the physiological implications of
 # proximal and distal drive as well as the general process used to articulate
 # a sequence of exogenous drive for simulating evoked responses, see the
-# `HNN ERP tutorial`_. Note that setting ``sync_within_trial=True`` creates
-# drives with synchronous input (arriving to and transmitted by hypothetical
-# granular cells at the center of the network) to all pyramidal and basket
-# cells that receive distal drive. Note that granule cells are not explicitly
-# modelled within HNN.
+# `HNN ERP tutorial`_. Note that setting ``n_drive_cells=1`` and
+# ``cell_specific=True`` creates a drive with synchronous input across cells
```

Comment:
You mean `cell_specific=False` ?

Reply:
Oops. Fixed in https://github.com/jonescompneurolab/hnn-core/pull/383/commits/2665da026cf90a4365a713e3ec08ad19d14f5528.